Netezos.Tests/Utils/Json/DateTimUtcTimezoneAppender.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Netezos.Tests.Forging
+{
+    // DateTime types are parsed without timezone but are assumed to be UTC.
+    // So append 'Z' when writing JSON DateTime values to accurately represent UTC.
+    class DateTimUtcTimezoneAppender : JsonConverter<DateTime>
+    {
+        public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return DateTime.Parse(reader.GetString());
+        }
+
+        public override void Write(Utf8JsonWriter writer, DateTime value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value.ToString("yyyy-MM-ddTHH:mm:ssZ"));
```

Comment:
The JSON files contain `Z` for UTC at the end of the date times. Since we deserialize to `DateTime` in the model, we don't have the timezone. I didn't want to change the model to be a `DateTimeOffset` (since we'd end up contending with timezones now), so I made a converter to append the `Z`.